rosdep/base.yaml

```diff
@@ -7371,6 +7400,7 @@ opencascade:
     stretch: [liboce-visualization-dev]
   fedora: [opencascade-devel]
   gentoo: [sci-libs/opencascade]
+  nixos: [opencascade]
```

Comment:
@kjeremy I was not able to find `opencascade` in the nixos index.
https://search.nixos.org/packages?channel=24.05&from=0&size=50&sort=relevance&type=packages&query=opencascade